packages/reakit/src/Composite/__utils/types.ts

```diff
@@ -10,4 +10,4 @@ export type Item = {
   disabled?: boolean;
 };
 
-export type Orientation = "horizontal" | "vertical";
+export type Orientation = "horizontal" | "vertical" | "both";
```

Comment:
I think we might need to run 💭
```
$ yarn keys
```
to prevent the type error below?  I'm not sure how package linking works on this project, but probably there's a way to get the updates to `reakit-system-bootstrap`. Maybe running the build scripts if the above doesn't work?
```
$ yarn build
```